neo4j-jdbc-bolt/src/main/java/org/neo4j/jdbc/bolt/BoltDriver.java

```diff
@@ -72,9 +77,21 @@ public BoltDriver() throws SQLException {
 	}
 
 	private AuthToken getAuthToken(Properties properties) {
-		if (properties.containsKey("user") && properties.containsKey("password")) {
+	    if(properties.isEmpty()) return AuthTokens.none();
+        //if (properties.containsKey("user") && properties.containsKey("password")) {
```

Comment:
@jexp because you don't have to pass AuthTokens.none if you don't provide the password or the user, but an basic one with wrong/missing credentials